cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/BatchSQLEngineAdapter.java

```diff
@@ -430,17 +430,17 @@ public Engine getSQLRelationalEngine() {
 
     CompletableFuture<T> future = new CompletableFuture<>();
 
-    Runnable joinTask = () -> {
+    Runnable runnable = () -> {
       try {
-        LOG.debug("Starting job for dataset '{}'", jobKey.getDatasetName());
+        LOG.debug("Starting {} job for dataset '{}'", jobKey.getJobType(), jobKey.getDatasetName());
         future.complete(jobFunction.get());
-        LOG.debug("Completed job for dataset '{}'", jobKey.getDatasetName());
+        LOG.debug("Started {} job for dataset '{}'", jobKey.getJobType(), jobKey.getDatasetName());
```

Comment:
Is it just started at this point or already job is done?

Reply:
Parameter says that it's "actual runnable that will do the work", so I'd assume it's done by now

Reply:
My bad, the actual jobs are completed by then. Push and Pull tasks have just "started" by the time we reach this log line.